boxcli/midcobra/telemetry.go

```diff
@@ -61,7 +63,7 @@ func (m *telemetryMiddleware) postRun(cmd *cobra.Command, args []string, runErr
 	if m.disabled {
 		return
 	}
-
+	initSentry(m.opts)
```

Comment:
Instead or re-using the existing telemetry "middleware", I lean towards creating a new one for sentry. Thoughts?

Or do you need to have it here in order to "link" the amplitude event to the sentry one via a sentry id?

Reply:
Need to have it here in order to "link"